interface-definitions/container.xml.in

```diff
@@ -334,6 +334,26 @@
                 </properties>
                 <defaultValue>rw</defaultValue>
               </leafNode>
+              <leafNode name="propagation">
+                <properties>
```

Comment:
There are more then those two options available in podman/docker.

```
shared, slave, private, unbindable, rshared, rslave, runbindable, or rprivate(default). See also mount(2).
```

We probably wan't to add them all?

Reply:
Done!